modules/sdk/project-templates-spring-mvc-portlet/src/main/resources/archetype-resources/src/main/webapp/WEB-INF/jsp/update.jsp

```diff
@@ -0,0 +1,66 @@
+<%--
```

Comment:
this jsp looks like it is for submitting a form to a service builder action.  But this one doesn't have that right?  Could we have just a simple post that posts to a portlet action and then render it simply?